tables/de-de-comp8.ctb

```diff
@@ -1,6 +1,7 @@
 
 #  Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
 #  Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com
+#  Copyright (C) 2012 by Aliminator83@gmail.com
```

Comment:
@taymun26 what is your real name?

Reply:
Is the 2012 correct?

Reply:
I think so, see b61536f97d0268a8fa57cbe0ce27cc347476a453

Reply:
Right, OK, just checking.